src/redux/actions/projects/loadProjects.js

```diff
@@ -63,9 +63,9 @@ const loadProjects = () => async (dispatch) => {
       // This section commented out because we going to use it
       // when samples loading is implemented in api v2
 
```

Comment:
can you remove this comment?

Reply:
I think it should be left because we are going to use it one we implement samples loading